api/tools/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+# This is needed so that Spring/Hibernate respects the Column name annotation
+# Without it, it will look for the snake cased version of the model's field name
+# We follow this convention is most places but it got tripped up on a field where
+# we did not (Dataset.prePackagedConceptSet)
```

Comment:
Should we plan on renaming that field for consistency's sake?

Reply:
filed https://precisionmedicineinitiative.atlassian.net/browse/RW-4062